Portfolio: validate contact form

Shewolf227 opened the pull request:
- The content of the email field has to be in lower case.
- You should implement the following interactions:
  - When the user submits the form, you check if the email is in lower case.
  - If the validation is OK, the form is sent.
  - If the validation is not OK, you show an error message to the user near the submit button informing them of the error and the form is not sent.

Shewolf227 committed:
Implement contact form validation

Add submit event listener to form

IshmaelKargbo committed:
Form validation in js

- When the user submits the form, you check if the email is in lower case.
- If the validation is OK, the form is sent.
- If the validation is not OK, you show an error message to the user near the submit button informing them of the error and the form is not sent.

Shewolf227 committed:
solve linter issues

Shewolf227 committed:
Update scripts.js

Shewolf227 committed:
Update scripts.js

Meltrust commented on css/styles.css:
```diff
@@ -102,6 +103,15 @@ html {
   margin: 12px 0;
 }
 
+#validation-msg {
```
- [ ] On your readme, kindly add the missing sections according to the [Microverse Readme Template](https://github.com/microverseinc/readme-template).  

  - Please, include the missing Acknowledgments section.
  - Please, include the missing Show your support section.
  - Please, include the missing Contributing section.
  - Please, include the missing License section.
 
## Explanation

Kindly remember that the Microverse Readme Template is required in almost all projects.   All sections are required except for the "Tests" and "Live demo" if not required by the project.

## Hints

Don't worry about having to make a huge readme on each project.   You can just do it once, and use it for all your projects (just customize the sections for each project).  

That way, your project will be well documented 💪

Meltrust reviewed (changes requested):
Hi @SoniaKarungi,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- Great implementation✔️ 

![You can do it](https://media.giphy.com/media/FemTGpEZeK27M5fKL4/giphy.gif)

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Shewolf227 committed:
implement feedback

- add Acknowledgement, contribution, licence and show your support sections to the ReadMe

Whoistolu reviewed (approved):
### Hi @SoniaKarungi 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
